src/transformers/models/layoutlm/modeling_tf_layoutlm.py

```diff
@@ -1382,6 +1382,7 @@ def call(
         Examples:
 
         ```python
+        >>> import tensorflow as tf
         >>> from transformers import LayoutLMTokenizer, TFLayoutLMForTokenClassification
         >>> import torch
```

Comment:
I think this can be removed too, no?

Reply:
Yes, great catch! Thanks. (I just focus on the line(s) where the exception occurs :-) )